ci

```diff
@@ -3,29 +3,30 @@
 set -ev # Ref https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps
 set -x
 
+OsLabel="ubuntu"
+if [ "${TRAVIS_OS_NAME:?}" = "osx" ]; then
+    OsLabel="osx-$(sw_vers -productVersion)"
+fi
 
-Version="$(cat VERSION)"
+Step=${1:?}
+Job=${2:?}
+BuildDir=${3:?}
+DeployEnv=${4:-test}
+Version=$(cat VERSION)
+BuiltPackagePath="${BuildDir}"/_build/prod/rel/epoch/epoch-"${Version}".tar.gz
```

Comment:
Please use `:?` for avoiding silent errors.

Reply:
i.e. `${Version:?}`

Reply:
And I would suggest referring to all vars with `${Var:?}` unless there is a good reason not to - otherwise updating the file becomes error prone.

Reply:
I'll make sure $Version is always valid. 

However in regards to other variables they all have guards during declaration e.g. BuildDir=${3:?} this guarantee that BuildDir is set and "non null" below in the script.

I intentionally removed ${Var:?} all over the script because it adds additional (useless IMO) noise in the script and decrease it's readability.

If you see some downsides or use cases about the above, please let me know so we can change it accordingly.

Reply:
If the next developer writes $BuildDIR (notice case change) will get silent error.

Reply:
Not a too strong opinion though - if not seen as common best practice no problem in doing as you suggest.

Reply:
done in 3682f50, let know if you see something missing